subsys/net/lib/lwm2m_client_utils/lwm2m/neighbour_cell_info.c

```diff
@@ -67,11 +67,21 @@ static void reset_signal_meas_object(uint16_t index)
 	lwm2m_engine_set_s32(path, 0);
 }
 
-static void update_signal_meas_objects(const struct lte_lc_cells_info *const cells)
+int lwm2m_update_signal_meas_objects(const struct lte_lc_cells_info *const cells)
 {
 	static int neighbours;
 	int i = 0;
 
+	if (cells == NULL) {
+		LOG_WRN("Invalid pointer");
```

Comment:
Isn't this an error that the developer has to fix?

Reply:
Yes. `LOG_ERR` is a better choice here.